pages/biosero_plate_state.vue

```diff
@@ -230,5 +231,6 @@ export default {
 #plate-table {
   min-width: 1200px;
   word-wrap: break-word;
+  table-layout: fixed;
```

Comment:
As only min-width is set and not width on the table and there is no width on the column, is the "table-layout: fixed" declaration actually a no-op? I think, at the moment, the table will just expand to max width possible. I am not sure if a computed width takes effect though. As there is no explicitly declared width on column widths, and word-wrap just rolls the text into the same cell, there should not be a clipping on the cell content based on the column widths of the first row of table.

https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout .